pages/keywords/index.page.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
```

Comment:
Some comments:

- keywords should be shorted alphabetically without any grouping (vocabulary is only a property)
- Avoid using a table. I provided an example using an unsorted list (bullet points). Plese check the issue
- For the links we should store links and titles. Please show the titles and create links with the URLs to provide a most friendly approach for showing urls.